### PR TITLE
[COST-3688] update celery task logs

### PR DESCRIPTION
**Pull request by maskarb**
## Jira Ticket

[COST-3688](https://issues.redhat.com/browse/COST-3688)

## Description

This change will update some of the celery task logs and add more context

### CODE REVIEW

File: koku/masu/processor/tasks.py [hunk 1]
```diff
@@ -566,6 +536,7 @@ def update_summary_tables(  # noqa: C901
 
     # Apply OCP on Cloud tasks
     if signature_list:
+        LOG.info(log_json(tracing_id, "chaining deletes and summaries"), report)
```
Comment (samdoran) [inline koku/masu/processor/tasks.py:539]: Is there an extra parenthesis here?
```suggestion
        LOG.info(log_json(tracing_id, "chaining deletes and summaries", report)
```

File: koku/masu/processor/tasks.py [hunk 2]
```diff
@@ -65,7 +65,7 @@
 from masu.util.oci.common import deduplicate_reports_for_oci
 
 
-LOG = logging.getLogger(__name__)
+LOG = get_task_logger(__name__)
```
Comment (samdoran) [inline koku/masu/processor/tasks.py:68]: Glad to see this used. I was surprised we don't use this currently for celery tasks.

File: koku/masu/external/kafka_msg_handler.py [hunk 3]
```diff
@@ -507,11 +507,12 @@ def summarize_manifest(report_meta, manifest_uuid):
     start_date = report_meta.get("start")
     end_date = report_meta.get("end")
 
-    context = {"account": report_meta.get("schema_name"), "provider_uuid": str(provider_uuid)}
+    context = {"account": schema_name, "provider_uuid": str(provider_uuid), "schema": schema_name}
```
[the remark this answers is not in the document]
Comment (maskarb) [reply koku/masu/external/kafka_msg_handler.py:510]: We will have to do that in a follow up. There are too many places right now to try and standardize on that.

File: koku/masu/processor/tasks.py (hunk 1 above)
Comment (maskarb) [reply koku/masu/processor/tasks.py]: Yea, there were a couple other places with this errant parenthesis. Good catch! I think I fixed them all.

File: koku/masu/processor/tasks.py [hunk 4]
```diff
@@ -419,45 +395,46 @@ def update_summary_tables(  # noqa: C901
         None
 
     """
-    if disable_summary_processing(schema_name):
-        LOG.info(f"Summary disabled for {schema_name}.")
+    schema = report.get("schema_name")
+    provider_type = report.get("provider_type")
+    provider_uuid = report.get("provider_uuid")
+    manifest_id = report.get("manifest_id")
```
[the remark this answers is not in the document]
Comment (maskarb) [reply koku/masu/processor/tasks.py]: I reverted this change.

File: koku/masu/processor/tasks.py [hunk 5]
```diff
@@ -368,31 +346,35 @@ def summarize_reports(reports_to_summarize, queue_name=None, manifest_list=None,
...
+                    invoice_month=month[2],
+                ).apply_async(queue=queue_name or UPDATE_SUMMARY_TABLES_QUEUE)
 
 
 @celery_app.task(name="masu.processor.tasks.update_summary_tables", queue=UPDATE_SUMMARY_TABLES_QUEUE)  # noqa: C901
 def update_summary_tables(  # noqa: C901
-    schema_name,
-    provider,
+    schema,
+    provider_type,
```
Comment (samdoran) [inline koku/masu/processor/tasks.py:377]: The doc string needs to be updated to match the parameter names.

File: koku/masu/processor/tasks.py [hunk 6]
```diff
@@ -466,29 +454,26 @@ def update_summary_tables(  # noqa: C901
...
         start_date, end_date = updater.update_summary_tables(
             start_date, end_date, tracing_id, invoice_month=invoice_month
         )
         if ocp_on_cloud:
             ocp_on_cloud_infra_map = updater.get_openshift_on_cloud_infra_map(start_date, end_date, tracing_id)
     except ReportSummaryUpdaterCloudError as ex:
-        LOG.info(
-            log_json(tracing_id, f"Failed to correlate OpenShift metrics for provider: {provider_uuid}. Error: {ex}")
-        )
+        LOG.info(log_json(tracing_id, f"failed to correlate OpenShift metrics: error: {ex}", context))
```
Comment (samdoran) [inline koku/masu/processor/tasks.py:476]: The str are repr of a `ReportSummaryUpdaterCloudError` exception is hopefully useful. If not, `traceback.format_exc()` will provide useful details.

**Review by samdoran — APPROVED** [review]

**Pull request merged**